app.js

```diff
@@ -0,0 +1,154 @@
+    let books;
+    if (localStorage.getItem("books") === null) {
+      books = [];
+    } else {
+      books = JSON.parse(localStorage.getItem("books"));
+    }
+    return books;
+  }
+
+  static addBook(book) {
+    const books = store.getBooks();
+
+    books.push(book);
+
+    localStorage.setItem("books", JSON.stringify(books));
+  }
+
+  static removeBook(authore) {
+    const books = store.getBooks();
+    books.forEach((book, index) => {
+      if (book.authore === authore) {
+        books.splice(index, 1);
+      }
+    });
+
+    localStorage.setItem("books", JSON.stringify(books));
+  }
+}
+
+//  Event for display books
+document.addEventListener("DOMContentLoaded", UI.displayBooks);
+
+// Add event listener to the add button
+document.querySelector("#form").addEventListener("submit", (e) => {
+  // Prevent actual submit
+  e.preventDefault();
+  const title = document.querySelector("#title").value;
+  const authore = document.querySelector("#author").value;
+
+  // Instatiate book
+  const book = new Book(title, authore);
+
+  // Add book data to screen
+  UI.addBookToList(book);
+
+  // Add book to store
+  store.addBook(book);
+
+  // clear fields
+  UI.clearField();
+});
+
+// Event to remove books
+document.querySelector("#data").addEventListener("click", (e) => {
+  // Remove book from UI
+  UI.deleteBook(e.target);
+  // Remove book from local storage
+  store.removeBook(e.target.parentElement.previousElementSibling.textContent);
+});
+
+// Navigation section start here
+document.getElementById("list").addEventListener("click", () => {
+  showList();
+});
+document.getElementById("add").addEventListener("click", () => {
+  showAdd();
+});
+document.getElementById("cont").addEventListener("click", () => {
+  showContact();
+});
+
+function defaultDisplay() {
+  document.getElementById("diplay-list").setAttribute("class", "non-show");
+  document.getElementById("input-data").setAttribute("class", "show");
+  document.getElementById("contact").setAttribute("class", "non-show");
+  document.getElementById("footer").setAttribute("class", "show");
+}
+defaultDisplay();
+
+function showList() {
+  document.getElementById("diplay-list").setAttribute("class", "show");
+  document.getElementById("input-data").setAttribute("class", "non-show");
+  document.getElementById("contact").setAttribute("class", "non-show");
+}
+
+function showAdd() {
+  document.getElementById("diplay-list").setAttribute("class", "non-show");
+  document.getElementById("input-data").setAttribute("class", "show");
+  document.getElementById("contact").setAttribute("class", "non-show");
+}
+
+function showContact() {
+  document.getElementById("diplay-list").setAttribute("class", "non-show");
+  document.getElementById("input-data").setAttribute("class", "non-show");
+  document.getElementById("contact").setAttribute("class", "show");
+}
+
+document.write(new Date().getFullYear());
+
+function ondate() {
+  document.querySelector(".date").innerHTML = Date();
+}
```

Comment:
- [ ] We want you to adhere to using _arrow_ syntax for functions in this activity. Kindly ensure to improve upon this aspect.